### DISCUSSION TITLE
A discord server for support & discussions

### DISCUSSION THREAD

**Discussion by joren-dev** [py-pdf/fpdf2#1157] [Ideas]
Would it be an idea to start a discord server and link that in the documentation, its getting pretty standard to have a discord server for projects like these and it'd be a great way to help each other and have discussions.

**Comment by Lucas-C** [comment]
Hi @joren-dev,

While I like the idea, I personally only connect to Discord once a month, and I don't have the time required to answer to people in any chat application. Moreover, I prefer to avoid the multiplicity of communication tools, which can quickly become a nightmare IMHO.
That's why I'll stick with GitHub issues & discussions for now, but other maintainers or even contributors are free to create such Discord server of course!

**Comment by gmischler** [comment]
I wholeheartedly agree with @Lucas-C on all points.
While I happen to have a Discord account, I only login there if I *really* have to, which hasn't been the case for several years now.
People can of course have discussions wherever they want, but let's keep the conversation relevant to the actual development activities focused in one place. All the communication tools that are necessary for the project can be found right here, and given the fairly low volume of interaction, moving parts of it to other places would only cause confusion.
